src/test/resources/com/fasterxml/jackson/integtest/gradle/build.gradle.kts

```diff
@@ -26,6 +26,10 @@ dependencies {
 
 repositories.mavenCentral()
 
+configurations.all {
+    resolutionStrategy.cacheDynamicVersionsFor(0, "seconds") // always refresh SNAPSHOTs
```

Comment:
I'll merge this, but forgot to ask: can be think of any case where short, but non-0 value would work well (avoid re-fetch on very short term)? Like, 5 or 10 second cache. Short enough to not be noticeable on interactive use, but maybe avoid re-fetch for CI?

If there is a potential benefit change as follow-up, but I'll merge this for now.

Thanks!

Reply:
> (avoid re-fetch on very short term)

No. It is not expensive to do. Only metadata is re-downloaded and not the Jars if the checksums did not change.